pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-inventory-storage</artifactId>
   <groupId>org.folio</groupId>
-  <version>14.1.0-SNAPSHOT</version>
+  <version>15.0.0-SNAPSHOT</version>
```

Comment:
We increment the version in a separate pull request when doing the release: https://dev.folio.org/guidelines/release-procedures/
Incrementing the version in a feature branch often results in a merge conflicts.